spec/models/manageiq/providers/vmware/infra_manager/refresher_spec.rb

```diff
@@ -1076,4 +1076,12 @@ def assert_relationship_tree
       }
     )
   end
+
+  context '.init_console' do
+    it "works" do
+      described_class.instance_variable_set(:@initialized_console, nil)
+      described_class.init_console
+      expect(described_class.instance_variable_get(:@initialized_console)).to be_truthy
```

Comment:
Could this create test contamination?  I'm wondering if we should clear the instance variable in an after block.

Reply:
you worried about the value. or the side effects of this metho?

just set to nil when done?

Reply:
yeah set to nil when done (in an after block to ensure it happens)...

memoized values at the class level are always a possible source of test contamination.